### PR TITLE
Feature/more shingles

### PR DESCRIPTION
**Pull request by Raycoms**
# Changes proposed in this pull request:
- Add 4 more types of shingles and shingleslabs
- (Sandstone, Stonebrick, nethe rand red nether)

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/blocks/BlockShingleSlab.java [hunk 1]
```diff
@@ -11,6 +12,7 @@
 import net.minecraft.util.math.AxisAlignedBB;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.IBlockAccess;
+import net.minecraftforge.registries.IForgeRegistry;
```
Comment (BuildTeamLinter) [inline src/main/java/com/minecolonies/coremod/blocks/BlockShingleSlab.java:15]: MINOR Remove this unused import 'net.minecraftforge.registries.IForgeRegistry'. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
      * References can be made to here.
      */
 
-    public static List<BlockTimberFrame> timberFrames = new ArrayList<>();
+    public static List<BlockTimberFrame>      timberFrames = new ArrayList<>();
```
Comment (BuildTeamLinter) [inline src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java:26]: MINOR Make this member "protected". [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2386)

**Comment by BuildTeamLinter** [comment]
SonarQube analysis reported 67 issues
* ![MAJOR][MAJOR] 59 major
* ![MINOR][MINOR] 2 minor
* ![INFO][INFO] 6 info

Watch the comments in this conversation to review them.

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [blocks#](https://github.com/Minecolonies/minecolonies/blob/c4702c57972f0d2b4304bb4cba22465658f6a033/src/main/java/com/minecolonies/coremod/blocks#): Add a 'package-info.java' file to document the 'blocks' package [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1228)
1. ![MAJOR][MAJOR] [BlockShingleSlab.java#L79](https://github.com/Minecolonies/minecolonies/blob/c4702c57972f0d2b4304bb4cba22465658f6a033/src/main/java/com/minecolonies/coremod/blocks/BlockShingleSlab.java#L79): Assign this magic number 255 to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [BlockShingleSlab.java#L129](https://github.com/Minecolonies/minecolonies/blob/c4702c57972f0d2b4304bb4cba22465658f6a033/src/main/java/com/minecolonies/coremod/blocks/BlockShingleSlab.java#L129): Reduce the number of returns of this method 17, down to the maximum allowed 6. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1142)
1. ![MAJOR][MAJOR] [BlockShingleSlab.java#L129](https://github.com/Minecolonies/minecolonies/blob/c4702c57972f0d2b4304bb4cba22465658f6a033/src/main/java/com/minecolonies/coremod/blocks/BlockShingleSlab.java#L129): The Cyclomatic Complexity of this method "getSlabShape" is 40 which is greater than 15 authorized. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
1. ![MAJOR][MAJOR] [BlockShingleSlab.java#L129](https://github.com/Minecolonies/minecolonies/blob/c4702c57972f0d2b4304bb4cba22465658f6a033/src/main/java/com/minecolonies/coremod/blocks/BlockShingleSlab.java#L129): Refactor this method to reduce its Cognitive Complexity from 34 to the 20 allowed. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)
1. ![MAJOR][MAJOR] [BlockShingleSlab.java#L167](https://github.com/Minecolonies/minecolonies/blob/c4702c57972f0d2b4304bb4cba22465658f6a033/src/main/java/com/minecolonies/coremod/blocks/BlockShingleSlab.java#L167): Assign this magic number 3 to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [BlockShingleSlab.java#L171](https://github.com/Minecolonies/minecolonies/blob/c4702c57972f0d2b4304bb4cba22465658f6a033/src/main/java/com/minecolonies/coremod/blocks/BlockShingleSlab.java#L171): Assign this magic number 3 to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [BlockShingleSlab.java#L175](https://github.com/Minecolonies/minecolonies/blob/c4702c57972f0d2b4304bb4cba22465658f6a033/src/main/java/com/minecolonies/coremod/blocks/BlockShingleSlab.java#L175): Assign this magic number 3 to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [BlockShingleSlab.java#L179](https://github.com/Minecolonies/minecolonies/blob/c4702c57972f0d2b4304bb4cba22465658f6a033/src/main/java/com/minecolonies/coremod/blocks/BlockShingleSlab.java#L179): Assign this magic number 3 to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)
1. ![MAJOR][MAJOR] [BlockShingleSlab.java#L183](https://github.com/Minecolonies/minecolonies/blob/c4702c57972f0d2b4304bb4cba22465658f6a033/src/main/java/com/minecolonies/coremod/blocks/BlockShingleSlab.java#L183): Assign this magic number 3 to a well-named constant, and use the constant instead. [rule](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by Raycoms** [comment]
@Asherslab is working on this, I'll close this pr therefore.

**Pull request closed without merging**